backend/src/agents/__init__.py

```
@@ -244,6 +253,29 @@ def init_backend(runtime: ToolRuntime, *, routes):
+    if DaytonaSandbox is None or Daytona is None:
+        return None, None
+
+    key = api_key or DAYTONA_API_KEY
+    if not key:
+        return None, None
+
+    try:
+        client = Daytona(DaytonaConfig(api_key=key))
+        sandbox = client.create()
+        backend = DaytonaSandbox(sandbox=sandbox)
+        return sandbox, backend
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)